docs/spec/api.md

```diff
@@ -1,7 +1,5 @@
 # Docker Registry HTTP API V2
 
-## Introduction
```

Comment:
I'll pull the api.md too.  Template was restored.  No wonder I was so confuzzled...it was the end of the day is my excuse for being so dense. :-D